client-js/README.md

```diff
@@ -6,140 +6,219 @@ This JavaScript client gives you an easy way to install and interact with the Ca
+      {
+        spender,
+        amount
+      },
+      5_000_000_000,
+      owner.publicKey,
       NETWORK_NAME,
-      [ownerAsymmetricKey] // Optional
+      [owner]
     );
-    ```
+```
+
+**Minting Tokens**
 
-  - Create a deploy to transfer from an account owner to a recipient given that the direct caller has been previously approved to spend the specified amount on behalf of the owner:
+Mint tokens and assign them to a recipient:
 
-    ```ts
-    const deploy = cep18.transferFrom(
+```ts
+const deploy = cep18.mint(
       {
-        owner: ownerPublicKey,
-        recipient: recipientPublicKey,
-        amount: transferAmount
+        owner: recipient,
+        amount
       },
       5_000_000_000,
-      approvedPublicKey,
+      owner.publicKey,
       NETWORK_NAME,
-      [approvedAsymmetricKey]
+      [owner]
     );
-    ```
+```
 
-- **Balances**
+**Burning Tokens**
 
-  Request the balance of an account with _balanceOf_:
+Burn tokens and reduce them from the owner's account:
 
-  ```ts
-  const balance = await cep18.balanceOf(publicKey);
-  ```
-
-- **Approvals**
-
-  Create a deploy to allow a spender to transfer up to a number of the direct caller’s tokens:
-
-  ```ts
-  const deploy = cep18.approve(
-    {
-      spender: spenderPublicKey,
-      amount: approveAmount
-    },
-    5_000_000_000,
-    ownerPublicKey,
-    NETWORK_NAME,
-    [ownerAsymmetricKey]
-  );
-  ```
+```ts
+const deploy = cep18.burn(
+      {
+        owner: recipient,
+        amount
+      },
+      5_000_000_000,
+      owner.publicKey,
+      NETWORK_NAME,
+      [owner]
+    );
+```
 
-- **Allowance**
+**Changing User Security**
 
-  Return the number of owner’s tokens allowed to be spent by spender:
+```ts
+const minterList = [ali.publicKey];
+const burnerList = [ali.publicKey, bob.publicKey];
+const deploy = cep18.changeSecurity(
+  {
+    adminList: [owner.publicKey],
+    minterList,
+    burnerList
+  },
+  5_000_000_000,
+  owner.publicKey,
+  NETWORK_NAME,
+  [owner]
+);
+```
 
-  ```ts
-  const allowance = await cep18.allowances(
-    ownersPublicKey,
-    spenderPublicKey
-  );
-  ```
 
 ## Event Handling
 
-CEP18 token supports [CES](https://github.com/make-software/casper-event-standard) and the token can be intalled with CES or NoCES method. If the token is installed with CES method, you can listen token events by using `EventStream` from `casper-js-sdk`. To consume token events, you should install `@make-software/ces-js-parser` by running
+The CEP-18 token supports the [Casper Event Standard (CES)](https://github.com/make-software/casper-event-standard), and the token can be installed with or without event logging as described [here](../cep18/README.md#eventsmode). If you install the token with the EventsMode set to CES, you can listen to token events using the `EventStream` from the `casper-js-sdk`. To consume token events, you should also install the `@make-software/ces-js-parser` by running this command:
```

Comment:
```suggestion
CEP-18 tokens support the [Casper Event Standard (CES)](https://github.com/make-software/casper-event-standard), and tokens can be installed with or without event logging as described [here](../cep18/README.md#eventsmode). If you install a token with the EventsMode set to CES, you can listen to token events using the `EventStream` from the `casper-js-sdk`. To consume token events, you should also install the `@make-software/ces-js-parser` by running this command:
```

Something about "The CEP-18 token supports the Casper Event Standard" feels off to me, and I can't quite put my finger on it.